$NetBSD$

Explicity flag library as static to match the rest of PrusaSlicer.
https://github.com/prusa3d/PrusaSlicer/pull/13896

--- src/slic3r-arrange/CMakeLists.txt.orig	2025-02-17 22:29:38.267707593 +0000
+++ src/slic3r-arrange/CMakeLists.txt
@@ -1,7 +1,7 @@
 project(slic3r-arrange)
 cmake_minimum_required(VERSION 3.13)
 
-add_library(slic3r-arrange
+add_library(slic3r-arrange STATIC
     include/arrange/Beds.hpp
     include/arrange/ArrangeItemTraits.hpp
     include/arrange/PackingContext.hpp
